src/hooks/useTvl.ts

```diff
@@ -27,8 +29,15 @@ export function useTvl(api: any) {
 
       const getTvl = async (): Promise<{ tvl: BN; tvlDefaultUnit: number }> => {
         const era = await apiRef.query.dappsStaking.currentEra();
-        const result = await apiRef.query.dappsStaking.eraRewardsAndStakes(era);
-        const tvl = result.unwrap().staked.valueOf();
+        let tvl;
+        if (isEnableIndividualClaim) {
```

Comment:
I feel like this can be made shorter. The code doesn't look clean

Reply:
[Fixed](https://github.com/AstarNetwork/astar-apps/pull/258/commits/ebd8c28a4c863e721825685de02a7485f3d7ce2d)